tutorials/transfers/index.md

```diff
@@ -0,0 +1,271 @@
+
+    ![Developer Tab](/images/tutorials/transfers/vested-transfers/vested-transfers-1.webp)
+
+2. Access the **vesting** pallet and select the **vestedTransfer** function:
+      1. Select the **account** from which you want to send the vested transfer
+      2. Choose the **vesting** pallet from the list of pallets 
+      3. Select the **vestedTransfer** function from the list of functions
+   
+    ![Vested Transfer Function](/images/tutorials/transfers/vested-transfers/vested-transfers-2.webp)
+
+3. Fill in the required fields:
+      1. **target** - the receiver's address
+      2. **schedule** - the vesting schedule. The schedule can be `linear` or `cliff`. The `linear` schedule will release the funds linearly over a period of time, while the `cliff` schedule will release the funds after a specific time period. 
+        - **locked** - the amount of funds that will be locked. It should be defined in [planks units](https://support.polkadot.network/support/solutions/articles/65000168663-how-many-planck-are-in-a-dot-){target=\_blank}
+         - **perBlock** - the amount of funds that will be released per block. In `cliff vesting`, the funds will be released all at once after the specified time period, so this value should be equal to the locked amount. In `linear vesting`, the funds will be released linearly over the specified time period, so this value should be calculated based on the total amount of funds and the duration of the vesting period.
+         - **startingBlock** - the block number when the funds will be released  
+       3. Click on the **Submit Transaction** button to proceed 
+
+    ![Vested Transfer Form](/images/tutorials/transfers/vested-transfers/vested-transfers-3.webp)
+
+    !!! note
+        The above image shows how to perform a cliff vesting transfer, since **locked** and **perBlock** are the same. If you want to perform a linear vesting transfer, you should calculate the **perBlock** value based on the total amount of funds and the duration of the vesting period. For example, if you want to release 100 DOT over a period of 10 blocks, the **perBlock** value should be 10 DOT.
+
+4. After submitting the transaction, a pop-up window will appear to confirm the transfer. Click on the **Sign and Submit** button to authorize the transaction:
+
+    ![Vested Transfer Confirmation](/images/tutorials/transfers/vested-transfers/vested-transfers-4.webp)
+
+5. To confirm that the transaction was successful, you can navigate to the **Accounts** tab and check the **locked** balances of the receiver's account (if the **startingBlock** has not been reached yet) or the **vested** balances (if the **startingBlock** has been reached):
+
+    ![Accounts Tab - Locked Balances](/images/tutorials/transfers/vested-transfers/vested-transfers-5.webp)
+
+#### Lazy Vesting
+
+Vesting is lazy, meaning that someone must explicitly call an extrinsic to update the lock placed on an account.
+
```

Comment:
I would add before the list of extrinsics an introduction saying something like there is a set of extrinsics that allow you to perform this operation and are defined in the vesting pallet:
- `vest` - ...
- `vest_other` - ...